### PR TITLE
initial vagrant box

### PR DESCRIPTION
**Pull request by geramirez**

### CODE REVIEW

**Comment by geramirez** [comment]
Works with `improve_importing` branch.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Getting Started
```
Comment (khandelwal) [inline README.md:4]: This readme doesn't tell me what this vagrant box does. What does it do? Why should I use it? How is it going to make my life easier?

Comment (geramirez) [inline README.md:4]: @khandelwal, k just updated the documentation.

**Comment by rjmajma** [comment]
Hey @khandelwal, can we merge this?

File: README.md (hunk 1 above)
Comment (khandelwal) [inline README.md:4]: What's Krang?

**Comment by khandelwal** [comment]
Not yet. The documentation needs a bit more work.

**Pull request merged**